Dockerfile

```diff
@@ -16,9 +15,13 @@ RUN mkdir -p ${TMP_DIR} && \
     # already create volume dir to later have correct rights
     mkdir ${TMP_DIR}/data
 
+RUN groupadd -g 1000 zeebe && \
+    adduser -u 1000 zeebe --system --ingroup zeebe
```

Comment:
Why do we need this in the builder image?

Reply:
To preserve permissions when copying over

Reply:
I don't think you need the user and groups, you can chown to whatever you want as far as I know

Reply:
I can try that :+1: